lib/ASTGen/Sources/ASTGen/ParameterClause.swift

```diff
@@ -178,19 +194,31 @@ extension ASTGenVisitor {
     functionParameterClause node: FunctionParameterClauseSyntax,
     for context: ParameterContext
   ) -> BridgedParameterList {
-    BridgedParameterList.createParsed(
+    var params: [BridgedParamDecl] = []
+    for (index, node) in node.parameters.enumerated() {
```

Comment:
Any reason not to use `map` here and below?

Reply:
🙄
* `node.parameters.enumerated().lazy.map { }` emits `LazyMapSequence<EnumeratedSequence<...>>` which does not conform to `LazyCollectionProtocol` where `bridgedArray(in:)` is declared.
* `node.parameters.lazy.enumerated().map { }` emits `Array<BridgedParamDecl>`